__tests__/utils.test.js

```diff
@@ -372,3 +352,68 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("Should update a specific article and return the articles update data", () => {
+    return request(app)
+      .patch("/api/articles/6")
+      .send({ inc_votes: 5 })
+      .expect(200)
+      .then((response) => {
+        const data = response.body.updatedArticle[0];
+        expect(data.article_id).toBe(6);
+        expect(data.hasOwnProperty("title")).toBe(true);
+        expect(data.hasOwnProperty("topic")).toBe(true);
+        expect(data.hasOwnProperty("author")).toBe(true);
+        expect(data.hasOwnProperty("body")).toBe(true);
+        expect(data.hasOwnProperty("created_at")).toBe(true);
+        expect(data.hasOwnProperty("votes")).toBe(true);
+        expect(data.hasOwnProperty("article_img_url")).toBe(true);
+      });
+  });
+
+  test("Should return a 404 article not found when given an id that does not exist", () => {
+    return request(app)
+      .patch("/api/articles/99")
+      .send({ inc_votes: 5 })
+      .expect(404)
+      .then((response) => {
+        const data = response.body.msg;
+        console.log(data);
+        expect(data).toBe("article not found");
+      });
+  });
+
+  test("Should return a 400 bad request when given an invalid id type", () => {
+    return request(app)
+      .patch("/api/articles/Kimiko")
+      .send({ inc_votes: 5 })
+      .expect(400)
+      .then((response) => {
+        const data = response.body.msg;
+        console.log(data);
+        expect(data).toBe("bad request");
+      });
+  });
+});
+
+//PLEASE IGNORE, THESE TESTS ARE BOT REQUIRED YET
```

Comment:
You're missing several tests here. What about no body, missing keys on the body, incorrect data type on the body, extra keys on the body?

You'll need to come back to this to solidly test this.